src/three-components/Renderer.js

```diff
@@ -58,6 +59,7 @@ export default class Renderer extends EventDispatcher {
     this.renderer.gammaFactor = GAMMA_FACTOR;
 
     this[$arRenderer] = ARRenderer.fromInlineRenderer(this);
+    this.textureManager = new TextureManager(this.renderer);
```

Comment:
This seems orthogonal to the `renderer` in all ways but lifetime. And the `renderer` is never disposed in practice (is that correct? should it be disposed under some set of circumstances?), so the texture manager won't be either.

What I'm driving at: is the texture manager relevant to the `renderer`, or the `environment` feature? Does it belong here?

Reply:
Correct, renderer is never disposed currently since all elements use the same renderer. We need one TextureGenerator-thing per Renderer, and textures, dispose-wise, are only used and GPU-cached for every WebGLRenderer that uses them

Reply:
We can split these concepts out later